tests/performance/jshint.js

```diff
@@ -5972,8 +5972,8 @@ var warnings = {
 	W006: "Confusing minuses.",
 	W007: "Confusing pluses.",
 	W008: "A leading decimal point can be confused with a dot: '{a}'.",
-	W009: "The array literal notation [] is preferrable.",
```

Comment:
Not sure if these should be left for legacy reasons

Reply:
Yes, it might be better to keep this one as is.

`tests/bench/large.js` as well.

Reply:
I've rebased this out